samples/Web/Startup.fs

```diff
@@ -94,16 +96,18 @@ type Startup() =
             | Memory _ ->
                 log.Fatal("Web App is using Volatile Store; Storage options: {options:l}", options)
                 Storage.MemoryStore.config (), log
+            | Mdb sp ->
```

Comment:
Just a thought; not an actual problem or anything to action unless you feel the desire and have the time...

Maybe its worth putting these in order consistently?

Cosmos
Dynamo
Esdb
Mdb
SSS ones

Given SSS is unmaintained, it's three can go at the end

I tend to put Memory either at the top or the bottom, but even strict alphabetical is at least more predictable

Please don't do this unless you think its important; such subjective cleanup can always be done as a follow-up exercise too